Lombiq.VueJs.Samples/ResourceManagementOptionsConfiguration.cs

```diff
@@ -30,6 +30,15 @@ static ResourceManagementOptionsConfiguration()
 
         // On the other hand make sure your .vue files are embedded during build, e.g. include this in the csproj:
         // <ItemGroup><EmbeddedResource Include="Assets\Scripts\VueComponents\*.vue" /></ItemGroup>
+
+        _manifest
+            .DefineSingleFileComponent(BusinessCard)
+            .SetDependencies(LoadingIndicator);
+
+        _manifest
+            .DefineSingleFileComponent(QrCard)
+            .SetDependencies(LoadingIndicator)
+            .SetDependencies(BusinessCard);
```

Comment:
This can be a single line:
```suggestion
            .SetDependencies(LoadingIndicator, BusinessCard);
```
![image](https://user-images.githubusercontent.com/164360/232741749-34027319-4725-4a69-9e74-d0c11b36e224.png)